servers/src/main/java/tachyon/master/journal/JournalTailerThread.java

```diff
@@ -80,9 +80,8 @@ public void shutdownAndJoin() {
     try {
       // Wait for the thread to finish.
       join();
-    } catch (InterruptedException ie) {
-      LOG.warn("{}: stopping the journal tailer caused exception: {}", mMaster.getName(),
-          ie.getMessage());
+    } catch (InterruptedException e) {
+      LOG.warn("{}: stopping the journal tailer caused an exception", mMaster.getName(), e);
```

Comment:
This way it will print the stacktrace - log4j has special handling when the last argument is an exception which isn't used in the format string.